IBAnimatable/MaskDesignable.swift

```diff
@@ -37,6 +39,11 @@ public extension MaskDesignable where Self: UIView {
     layer.cornerRadius = ceil(min(bounds.width, bounds.height))/2
   }
   
+  public func maskPolygon() {
+    let polygonPath = maskPolygonBezierPath(Int(arc4random() % 10 + 3))
```

Comment:
Definitely not, but until we can customise the parameters, it's the an easy way to create polygon.